source/docs/software/wpilib-tools/robot-simulation/unit-testing.rst

```diff
@@ -0,0 +1,227 @@
+Unit Testing
+============
+
+Unit testing is a method of testing code by dividing the code into the smallest "units" possible and testing each unit. In robot code, this can mean testing the code for each subsystem individually. There are many unit testing frameworks for most languages. Java robot projects have `JUnit 4 <https://junit.org/junit4/>`__ available by default, and C++ robot projects have `Google Test <https://github.com/google/googletest/blob/master/docs/primer.md>`__.
+
+Writing Testable Code
+^^^^^^^^^^^^^^^^^^^^^
+
+Our subsystem will be an Infinite Recharge intake mechanism containing a piston and a motor: the piston deploys/retracts the intake, and the motor will pull the Power Cells inside. Since it won't do anything, we don't want the motor to run if the intake mechanism isn't deployed.
```

Comment:
Reword the last sentence of this paragraph to "We don't want the motor to run if the intake mechanism isn't deployed because it won't do anything."